snmp/datadog_checks/snmp/data/default_profiles/_palo-alto.yaml

```diff
@@ -0,0 +1,10 @@
+extends:
+  - _base.yaml
+  - _generic-host-resources-base.yaml
+  - _generic-if.yaml
```

Comment:
I think it might be safer to not have:
```
  - _generic-host-resources-base.yaml
  - _generic-if.yaml
```
here.

But have leaf profiles extend then directly.

The rational is that, it we have in the future a new palo alto profile that shouldn't extend then, it will be harder to deal with.